HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1571,46 +1748,209 @@ def self.get_cooling_setpoints(hpxml_bldg, hvac_control, has_ceiling_fan, year,
     return clg_wd_setpoints, clg_we_setpoints
   end
 
-  # TODO
+  # Geothermal heat pump cooling capacity ratios for advanced model (variable-speed coils)
   #
   # @param heat_pump [HPXML::HeatPump] The HPXML heat pump of interest
-  # @return [nil]
-  def self.set_curves_gshp(heat_pump)
-    hp_ap = heat_pump.additional_properties
-
-    # E+ equation fit coil coefficients generated following approach in Tang's thesis:
-    # See Appendix B of  https://shareok.org/bitstream/handle/11244/10075/Tang_okstate_0664M_1318.pdf?sequence=1&isAllowed=y
-    # Coefficients generated by catalog data: https://files.climatemaster.com/Genesis-GS-Series-Product-Catalog.pdf, p180
-    # Data point taken as rated condition:
-    # EWT: 80F EAT:80/67F, AFR: 1200cfm, WFR: 4.5gpm
+  # @return [Array<Double, Double>] Cooling apacity ratios at each speed
+  def self.get_cool_capacity_ratios_gshp(heat_pump)
+    # For each speed, ratio of capacity to nominal capacity
```

Comment:
I'm hoping we can remove the `get_cool_capacity_ratios_gshp`, `get_heat_capacity_ratios_gshp`, and `set_curves_gshp` methods here, and integrate all of that code into the `set_hvac_cooling_performance` and `set_hvac_heating_performance` methods in defaults.rb. Then air-source and ground-source systems will be pretty consistent.

It'd be nice to move the `set_gshp_assumptions` into defaults.rb too. And probably it should be renamed to `set_geothermal_loop_assumptions`, it seems to be specific to the geothermal loop and not the GHP.

Reply:
I got the point that you moved those methods for ashp to be in defaults.rb when resolving conflicts, but found the `set_curves_gshp` still there, not sure if it's intended because it's not using table lookup, or if it's something you want to resolve when pulling in the GHP work. I agree that it's better to be in defaults.rb, I'll give it a try.